app/stores/AppStore.js

```diff
@@ -50,6 +50,11 @@ export default class AppStore extends Store {
     return 'en-US'; // default
   }
 
+  @computed get termsOfUse(): string {
+    const localizedTermsOfUse = require(`../i18n/locales/terms-of-use/${this.currentLocale}.md`); // eslint-disable-line
```

Comment:
What was the resolution for this problem? Is this (still) a problem or does it work on all OS?